daisy/workflow/step_create_disks_test.go

```diff
@@ -138,8 +139,20 @@ func TestCreateDisksValidate(t *testing.T) {
 	ctx := context.Background()
 	// Set up.
 	w := testWorkflow()
-	w.ComputeClient = nil
-	w.StorageClient = nil
+	_, c, err := daisyCompute.NewTestClient(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I'd suggest faking the TestClient's GetProject/GetZone/GetMachine methods, if that is easier than writing request handlers for you.

My thoughts: the compute client unit tests should understand and test the requests, but CreateDisks (for example) really only cares about the GetProject and GetZone interface.